docs/navigation-actions.md

```diff
@@ -67,6 +67,8 @@ When dispatching `setParams`, the router will produce a new state that has chang
 * `params` - _object_ - required - New params to be merged into existing route params
 * `key` - _string_ - required - Route key that should get the new params
 
+#TODO - idk
```

Comment:
It's now the `BaseActions` exported from `/core`. We can rename it to `NavigationActions`

Reply:
common